src/fast_check/transform.rs

```diff
@@ -229,7 +232,7 @@ impl<'a> FastCheckTransformer<'a> {
     comments: &mut CommentsMut,
     is_ambient: bool,
   ) -> Result<Vec<ModuleItem>, Vec<FastCheckDiagnostic>> {
-    let mut final_body = Vec::with_capacity(body.len());
+    let mut final_body = vec![];
```

Comment:
Maybe `body.len() + self.public_inferred_namespaces.iter().map(|v| v.len()).count()`.

Reply:
Nope. This is a bad suggestion because the public_inferred_namespaces will be empty here.